tests/utils.js

```diff
@@ -60,18 +68,26 @@ exports.withTestMapperServer = (port, test) => {
     // This is important to be able to create endpoints during tests and post data to them.
     process.env.DYNAMIC_ENDPOINTS = true
 
-    const testMapperServer = await spawnServer({
+    await spawnServer({
       SERVER_PORT: port,
       DYNAMIC_ENDPOINTS: true
     })
+      .then(async testMapperServer => {
+        // Execute the tests
+        await test(t)
 
-    // Execute the tests
-    await test(t)
+        // Close the server instance after tests
+        t.teardown(async () => {
+          testMapperServer.kill('SIGINT', 0)
```

Comment:
@MattyJ007, doesn't the kill function only have one argument?

Reply:
Doesn't need any parameters...